app/controllers/comments_controller.rb

```diff
@@ -13,6 +13,30 @@ def create
     end
   end
 
+  def edit
+    @comment = Comment.find(params[:id])
+  end
+
+  def update
+    @comment = Comment.find(params[:id])
+    if @comment.update(comment_params)
+      redirect_to review_path(@comment.review),
+                  notice: "Comment has been updated"
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @comment = Comment.find(params[:id])
+    if @comment.destroy
+      redirect_to review_path(@comment.review),
+                  notice: "Comment has been deleted"
+    else
+      render "reviews/show"
```

Comment:
I would add a flash notice to let the user know that the comment wasn't deleted.